__tests__/app.test.js

```diff
@@ -30,21 +29,34 @@ describe('* invalid endpoints', () => {
         .get('/doesn"t/exist')
         .expect(404)
         .then((res) => {
-            console.log(res.body.msg)
             expect(res.body.msg).toEqual('Endpoint not found')
-
+            
         })
     })
-
+    
     it('returns a 404 status code when making an invalid put request', () => {
         return request(app)
         .get('/doesn"t/exist')
         .send({msg: 'This should message should never '})
-        .expect(404)
         .then((res) => {
-            console.log(res.body.msg)
             expect(res.body.msg).toEqual('Endpoint not found')
-
+            
         })
     })
 })
+
+describe('GET /api', () => {
+    it('returns a 200 status code and returns an object of available end points', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((res) => {
```

Comment:
destructuring streamlines syntax:

.then( { body }) => { 
const endpoints = body.newEndpoints

or even one step further

.then( { body: {endpoints} } ) => { 

and the endpoints is immediately available